packages/javascript/bh-shared-ui/src/components/ExploreTable/ExploreTableDataCell/ExploreTableDataCell.test.tsx

```diff
@@ -48,4 +49,18 @@ describe('ExploreTableDataCell', () => {
         const copyButton = await screen.findByText('copy');
         expect(copyButton).toBeInTheDocument();
     });
+
+    const ADTimeProperties = ['lastlogon', 'lastlogontimestamp'];
```

Comment:
funny thing about the `ADSpecificTimeProperties` is that the first and last value (`whencreated`, `pwdlastset`) are considered `CommonKindProperties` and wont trigger the ad formatting. Unpacking that feels out of scope for this ticket as I just need the explore table cells to match entity panel values and both will go thru this same codeflow.